wordpress-mobile/WordPress-Android
# Run connected tests nightly on Firebase Test Lab

Now that https://github.com/wordpress-mobile/WordPress-Android/pull/9835 is merged, we can start running our now mocked connected tests on CI. This change runs all connected tests on Firebase Test Lab at midnight each night. It runs the same tests as `./gradlew WordPress:connectedVanillaDebugAndroidTest` and includes all e2e tests and the screenshot tests.

As agreed with @loremattei we will start by running them on a nightly basis and can later move them to every PR when we feel they are reliable and useful. We can also add Slack notifications on failure when we feel we can rely on them.

To test:

- See that the connected tests ran successfully [here](https://circleci.com/gh/wordpress-mobile/WordPress-Android/12922).

Update release notes:
